src/current/molt/molt-fetch.md

```diff
@@ -503,6 +504,92 @@ The JSON is formatted as follows:
 - `column` specifies the column that will use the custom type mapping in `type-kv`. If `*` is specified, then all columns in the `table` with the matching `source-type` are converted.
 - `type-kv` specifies the `source-type` that maps to the target `crdb-type`.
 
+### Transformations
+
+You can define transformation rules to be performed on the target schema during the fetch process. These can be used to:
+
+- Map [computed columns]({% link {{ site.current_cloud_version }}/computed-columns.md %}) to a target schema.
+- Map [partitioned tables]({% link {{ site.current_cloud_version }}/partitioning.md %}) to a single target table.
+- Rename tables on the target schema.
+
+Transformation rules are defined in the JSON file indicated by the `--transformations-file` flag. The JSON is formatted as follows:
```

Comment:
The code snippet contains examples of transformation rules. The rules are formatted in JSON. 

```suggestion
Transformation rules are defined in the JSON file indicated by the `--transformations-file` flag. Two examples of rules in JSON format follow:
```